### PR TITLE
Updated  futuresPositionRisk URL to V2. Added futuresGetOrder, futuresLeverage method.

### PR DESCRIPTION
**Pull request by anwebdevelopers**
1) Changed futuresPositionRisk URL to V2 in accordance with the binance documentation https://binance-docs.github.io/apidocs/futures/en/#position-information-v2-user_data
2) Added futuresGetOrder method to check an order's status.
3) Added futuresLeverage method.
4) Added description to the README

### CODE REVIEW

**Comment by balthazar** [comment]
Thanks! Do you mind adding the new methods to the README?

**Comment by anwebdevelopers** [comment]
> Thanks! Do you mind adding the new methods to the README?

Yes, I will definitely do it a little later, when time becomes free.

**Comment by anwebdevelopers** [comment]
Added description to the README

File: README.md [hunk 1]
```diff
@@ -1648,6 +1650,108 @@ console.log(await client.capitalDepositAddress({ coin: 'NEO' }))
 
 ### Futures Authenticated REST endpoints
 
+#### futuresGetOrder
+
+Check an order's status.
+
+##### Weight: 1
```
Comment (balthazar) [inline README.md:1657]: Hum you should remove this subtitle

File: README.md [hunk 2]
```diff
@@ -1648,6 +1650,108 @@ console.log(await client.capitalDepositAddress({ coin: 'NEO' }))
...
+
+- These orders will not be found
+  - order status is CANCELED or EXPIRED, <b>AND</b>
+  - order has NO filled trade, <b>AND</b>
+  - created time + 7 days < current time
+  
+##### Parameters:
```
Comment (balthazar) [inline README.md:1664]: Same goes for all these actually, we're not using that so I'd rather keep it consistent

File: README.md [hunk 3]
```diff
@@ -1648,6 +1650,108 @@ console.log(await client.capitalDepositAddress({ coin: 'NEO' }))
...
+
+| Name              | Type   | Mandatory | Description      |
+| ----------------- | ------ | --------  | ---------------- |
+| symbol            | STRING | YES       | The pair name    |
+| orderId           | LONG   | NO        |                  |
+| origClientOrderId | STRING | NO        |                  |
+| recvWindow        | LONG   | NO        |                  |
+
+##### Notes:
```
Comment (balthazar) [inline README.md:1673]: ditto

File: README.md [hunk 4]
```diff
@@ -1648,6 +1650,108 @@ console.log(await client.capitalDepositAddress({ coin: 'NEO' }))
...
+}
+```
+</details>
+
+
+#### futuresLeverage
+
+Change user's initial leverage of specific symbol market.
+
+##### Weight: 1
```
Comment (balthazar) [inline README.md:1724]: ditto

File: README.md [hunk 5]
```diff
@@ -1648,6 +1650,108 @@ console.log(await client.capitalDepositAddress({ coin: 'NEO' }))
...
+
+##### Parameters:
```
Comment (balthazar) [inline README.md:1726]: ditto

**Comment by balthazar** [comment]
Thanks for the PR and the changes!

**Pull request merged**